External/Utilities/Opacities/Bruenn85/Makefile_CreateTable

```diff
@@ -1,4 +1,4 @@
-include ../../../../Build/Makefile_Build
+include $(WEAKLIB_DIR)/External/Build/Makefile_Build
```

Comment:
Is `WEAKLIB_DIR` always defined by the user?
this may need another line above it like:
```
WEAKLIBDIR ?= ../../../..
```